scripts/roles/todd/handlers/main.yml

```diff
@@ -0,0 +1,4 @@
+---
+  - include_vars: "{{ansible_os_family}}.yml"

```

Comment:
This was an accidental inclusion from the role I used as a "starter" for this. Removed in 566fb9ff02e3ef75295489f79689c5b322849268